chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -173,23 +175,33 @@ pane.collections.library		= My Library
 pane.collections.publications  = My Publications
 pane.collections.feeds  = Feeds
 pane.collections.groupLibraries	= Group Libraries
+pane.collections.feedLibraries      = Feeds
 pane.collections.trash			= Trash
 pane.collections.untitled		= Untitled
 pane.collections.unfiled		= Unfiled Items
 pane.collections.duplicate		= Duplicate Items
 
 pane.collections.menu.rename.collection			= Rename Collection…
-pane.collections.menu.edit.savedSearch			= Edit Saved Search
+pane.collections.menu.edit.savedSearch			= Edit Saved Search…
+pane.collections.menu.edit.feed					= Edit Feed…
 pane.collections.menu.delete.collection			= Delete Collection…
 pane.collections.menu.delete.collectionAndItems	= Delete Collection and Items…
 pane.collections.menu.delete.savedSearch			= Delete Saved Search…
+pane.collections.menu.delete.feedAndItems			= Delete Feed and Items…
 pane.collections.menu.export.collection			= Export Collection…
 pane.collections.menu.export.savedSearch			= Export Saved Search…
+pane.collections.menu.export.feed			= Export Feed…
 pane.collections.menu.createBib.collection		= Create Bibliography From Collection…
 pane.collections.menu.createBib.savedSearch		= Create Bibliography From Saved Search…
+pane.collections.menu.createBib.feed		= Create Bibliography From Feed…
 
 pane.collections.menu.generateReport.collection	= Generate Report from Collection…
 pane.collections.menu.generateReport.savedSearch	= Generate Report from Saved Search…
+pane.collections.menu.generateReport.feed	= Generate Report from Feed…
+
+pane.collections.menu.refresh.feed			= Refresh Feed
+pane.collections.menu.toggleRead.markRead	= Mark Read
+pane.collections.menu.toggleRead.markUnread	= Mark Unread
```

Comment:
"Mark as Read" and "Mark as Unread" are a bit more standard. (Can change the labels to those too.)

Can also drop the `toggleRead`, but these don't really belong here anyway, since they're used in the items list too (and apply more to the items). I'd go with `pane.items.markAsRead`.